test/code-build-test.js

```diff
@@ -104,6 +104,21 @@ describe("githubInputs", () => {
       .and.to.deep.equal(["one", "two", "three", "four"]);
   });
 
+  it("skips override when parameter is set to true", () => {
```

Comment:
this test case name is a bit confusing. should we just called it `can process disable-source-override `